overlord/devicestate/devicemgr.go

```diff
@@ -309,6 +309,7 @@ func (m *DeviceManager) StartUp() error {
 }
 
 func (m *DeviceManager) shouldMountUbuntuSave() bool {
+	// TODO: this should check DeviceCtx.IsClassicBoot() instead
```

Comment:
I need to double check on this

Reply:
This was a bit of a drive-by comment, without something here the change 2  that creates the device key will fail with something like `"internal error: cannot access device keypair manager if ubuntu-save is unavailable"` (from memory).